specs/monitoring/common/schemas/Incidents.yml

```
@@ -0,0 +1,12 @@
+title: incidents
+description: List of incidents reported for this cluster.
+type: array
+items:
+  type: object
+  properties:
+    t:
+      type: integer
+      format: int64
+      description: Timestamp of the incident report.
```

```suggestion
      description: Timestamp of the incident report.
```
For this and other times, what's the timestamp format? Unix epoch time in seconds?